RELEASE/scripts/sl_ascend.ash

```diff
@@ -12703,8 +12704,57 @@ boolean L9_chasmBuild()
 		return true;
 	}
 
+
+	// -Combat is useless here since NC is triggered by killing Orcs...So we kill orcs better!
+	asdonBuff($effect[Driving Intimidatingly]);
+
+	// Check our Load out to see if spells are the best option for Orc-Thumping
+	if(setFlavour($element[cold]) && canUse($skill[Stuffed Mortar Shell]))
+	{
+		set_property("sl_useSpellsInOrcCamp", true);
+	}
```

Comment:
I don't think this belongs in a property since it's not used outside of this scope. You can use a boolean or better yet, a skill.

```
skill orcCampSpell = $skill[none];
if(setFlavour($element[cold]) && canUse($skill[Stuffed Mortar Shell])) orcCampSpell = $skill[Stuffed Mortar Shell];
else if (...)...

if(orcCampSpell != $skill[none])
{
  // maximize cold spell damage
}
else
{
  // maximize weapon cold dmg, attack
}
```

Reply:
I see your point on it not being used outside of this scope.

Interesting idea on the skill. Done.